protocols/kad/src/record.rs

```diff
@@ -25,10 +25,14 @@ pub mod store;
 use bytes::Bytes;
 use instant::Instant;
 use libp2p_core::{multihash::Multihash, Multiaddr, PeerId};
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
 use std::borrow::Borrow;
 use std::hash::{Hash, Hasher};
 
 /// The (opaque) key of a record.
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "serde", serde(crate = "_serde"))]
```

Comment:
I am missing something here. Why is this line needed in `libp2p-kad` but not in `libp2p-gossipsub`?

Reply:
:pray: Thank you @laptou. That makes sense.